Getting-Started-with-Akka-http-signature.md

```diff
@@ -114,111 +114,77 @@ This will save the contents of the key in String format within the file found by
 
 ### Saving a public Key to RDF
 
-A user can also transform his public keys to an RDF. In order to do this one must first import multiple files in order to resolve the required dependencies.
+A user can also transform his public keys to an RDF. In order to do this one must first import multiple files in order to resolve the required dependencies:
 
 The org.w3 declarations required are:
 
 ```scala
-import org.w3.banana.binder
-import org.w3.banana.binder.RecordBinder
-implicit val binder = RecordBinder
-val cert = CertPrefix[Rdf]
-```
-Several java imports from java's security and math library are also required for this process:
+import $ivy.`run.cosy::solid-client:0.1-SNAPSHOT`
+import run.cosy.auth.RSAKeys
+import com.typesafe.sslconfig.akka._
+import akka.actor._
+import akka.http.scaladsl.model.Uri
+import com.typesafe.config.ConfigFactory
+import akka.event.Logging
+import akka.stream.{ActorMaterializer, ActorMaterializerSettings,Supervision,_}
+import akka.http.scaladsl.model.{Uri=>AkkaUri,_}
 
-```scala
-import java.math.BigInteger
-import java.security.KeyFactory
-import java.security.interfaces.RSAPublicKey
-import java.security.spec.RSAPublicKeySpec
-```
+import scala.concurrent.ExecutionContext
+
+import org.w3.banana.jena.Jena
 
-After that the binder and all it's functionalities can be imported:
+import Jena._
 
-```scala
-import org.w3.banana.binder._
-import recordBinder._
+import run.cosy.solid.client.Web._
+import java.security.interfaces.RSAPublicKey
 ```
 
-Finally, after that, the user can create the Cert object that contains the required dependencies to turn a Key into a Pointed Graph
+Finally, after that, the user can create a value for the public key String collected from the .pem file in located within the .keys directory.
```

Comment:
horrible grammar. A value? 
The thing to point out here is that different software could be using this public and private keys on different days, and so need to read this in order to be able to authenticate to different resources on the web. So we need to de-serialise a.k.a [marshal](http://doc.akka.io/docs/akka-http/10.0.9/scala/http/common/unmarshalling.html#unmarshalling) the public key. Those are all terms you should learn of, and use correctly.